quelpa.el

```diff
@@ -371,9 +422,14 @@ If there is an error and no existing checkout return nil."
 (defun quelpa-get-melpa-recipe (name)
   "Read recipe with NAME for melpa git checkout.
 Return the recipe if it exists, otherwise nil."
-  (let* ((recipes-path (expand-file-name "package-build/recipes" quelpa-build-dir))
+  (let* ((recipes-path (car quelpa-melpa-recipe-dirs))
          (files (directory-files recipes-path nil "^[^\.]+"))
          (file (assoc-string name files)))
```

Comment:
it would be better to rewrite this, not previously "letting" all the variables.

with dolist you don't need to do that, example:

```cl
  (dolist (hook '(emacs-lisp-mode-hook lisp-interaction-mode-hook))
    (add-hook hook 'my-company-elisp-setup)))
```

using `setq` is discouraged (sometimes you can't get around it though)... that's procedural-style. whenever possible try to use the functional approach (working with map etc.)